src/main/java/com/basho/riak/client/core/query/timeseries/ColumnDescription.java

```diff
@@ -0,0 +1,75 @@
+package com.basho.riak.client.core.query.timeseries;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ *
+ * @author Alex Moore <amoore at basho dot com>
+ * @since 2.0.3
+ */
+public class ColumnDescription
```

Comment:
it might be a good idea to add proper equals() and hashCode()  methods as well as toString()